### ISSUE TITLE
Learning rate scheduler is instantiated but never used in train_puffin_D.py

### ISSUE THREAD

**Issue by CHNFTQ** [jzhoulab/puffin_manuscript#3]
In [train/train_puffin_D.py](https://github.com/jzhoulab/puffin_manuscript/blob/main/train/train_puffin_D.py), the ReduceLROnPlateau scheduler is created with:
```python
scheduler = ReduceLROnPlateau(optimizer, mode='max', factor=0.9, patience=10, threshold=0)
```

However, it looks like scheduler.step(...) is never called in the training loop, so the scheduler doesn't seem to have any effect.

Just wanted to check — is this intentional, or was the scheduler meant to be used during training?

**Comment by jzthree** [comment]
We indeed are not using the scheduler, and this was left over from older code. This line can be safely deleted.

**Comment by CHNFTQ** [comment]
Thank you for clarifying!
